### ISSUE TITLE
can‘t download 

### ISSUE THREAD

**Issue by iOX1D** [yuliskov/SmartTube#3168]
kutt.it seems to be down, i get forwarded to cloudflare saying Host Error

**Comment by TroglodyteGuy** [comment]
Same, trying to install on my Google TV device, and it returns “Error: No connection”.

I am currently unable to install stn (beta or stable).

**Comment by zepelin16** [comment]
Same with ATV Bridge SmartTube app: https://kutt.it/stn_bridge_atv

**Comment by LunarV2** [comment]
yeah, can't access it at all, however, when i do `ping kutt.it` it still gives a response

**Comment by LunarV2** [comment]
kutt.it is now online, issue solved.
